diff --git a/node_modules/react-video-recorder/lib/defaults/error-view.js b/node_modules/react-video-recorder/lib/defaults/error-view.js
index 6c81d20..9102b64 100644
--- a/node_modules/react-video-recorder/lib/defaults/error-view.js
+++ b/node_modules/react-video-recorder/lib/defaults/error-view.js
@@ -10,7 +10,7 @@ var _react = _interopRequireDefault(require("react"));
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
 
 var ErrorView = function ErrorView() {
-  return _react["default"].createElement("div", null, "Oh snap! Your browser failed to record your video.", _react["default"].createElement("br", null), _react["default"].createElement("br", null), "Please restart it and try again \uD83D\uDC4D");
+  return _react["default"].createElement("div", null, "Oh não! Seu navegador falhou ao gravar o vídeo.", _react["default"].createElement("br", null), _react["default"].createElement("br", null), "Reinicie o navegador e tente novamente \uD83D\uDC4D");
 };
 
 var _default = ErrorView;
diff --git a/node_modules/react-video-recorder/lib/defaults/loading-view.js b/node_modules/react-video-recorder/lib/defaults/loading-view.js
index b2f4259..70c072d 100644
--- a/node_modules/react-video-recorder/lib/defaults/loading-view.js
+++ b/node_modules/react-video-recorder/lib/defaults/loading-view.js
@@ -17,7 +17,7 @@ var LoadingMessage = _styledComponents["default"].div.withConfig({
 })(["font-family:Arial;"]);
 
 var LoadingView = function LoadingView() {
-  return _react["default"].createElement(LoadingMessage, null, "Loading...");
+  return _react["default"].createElement(LoadingMessage, null, "Carregando...");
 };
 
 var _default = LoadingView;
diff --git a/node_modules/react-video-recorder/lib/defaults/record-button.js b/node_modules/react-video-recorder/lib/defaults/record-button.js
index 411cef1..b70ed18 100644
--- a/node_modules/react-video-recorder/lib/defaults/record-button.js
+++ b/node_modules/react-video-recorder/lib/defaults/record-button.js
@@ -46,7 +46,7 @@ Button.defaultProps = {
 };
 
 var _default = function _default(props) {
-  return _react["default"].createElement(RecWrapper, null, _react["default"].createElement(Instructions, null, _react["default"].createElement("div", null, "PRESS "), _react["default"].createElement(InstuctionsHighlight, null, " REC "), "WHEN READY"), _react["default"].createElement(ButtonBorder, null, _react["default"].createElement(Button, props)));
+  return _react["default"].createElement(RecWrapper, null, _react["default"].createElement(Instructions, null, _react["default"].createElement("div", null, "APERTE O "), _react["default"].createElement(InstuctionsHighlight, null, " BOTÃO "), "PARA GRAVAR"), _react["default"].createElement(ButtonBorder, null, _react["default"].createElement(Button, props)));
 };
 
 exports["default"] = _default;
\ No newline at end of file
diff --git a/node_modules/react-video-recorder/lib/defaults/render-actions.js b/node_modules/react-video-recorder/lib/defaults/render-actions.js
index 58774c1..b5ef713 100644
--- a/node_modules/react-video-recorder/lib/defaults/render-actions.js
+++ b/node_modules/react-video-recorder/lib/defaults/render-actions.js
@@ -64,7 +64,7 @@ var Actions = function Actions(_ref) {
       return _react["default"].createElement(_button["default"], {
         onClick: onStopReplaying,
         "data-qa": "start-replaying"
-      }, "Use another video");
+      }, "Usar outro vídeo");
     }
 
     if (isRecording) {
diff --git a/node_modules/react-video-recorder/lib/defaults/unsupported-view.js b/node_modules/react-video-recorder/lib/defaults/unsupported-view.js
index 970e67d..b944726 100644
--- a/node_modules/react-video-recorder/lib/defaults/unsupported-view.js
+++ b/node_modules/react-video-recorder/lib/defaults/unsupported-view.js
@@ -10,7 +10,7 @@ var _react = _interopRequireDefault(require("react"));
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
 
 var UnsupportedView = function UnsupportedView() {
-  return _react["default"].createElement("div", null, "This browser is uncapable of recording video");
+  return _react["default"].createElement("div", null, "Não é possível gravar vídeo nesse navegador");
 };
 
 var _default = UnsupportedView;
